bin/crew

```diff
@@ -869,11 +869,18 @@ end
 
 def compress_doc(dir)
   # check whether crew should compress
-  return if CREW_NOT_COMPRESS || !File.exist?("#{CREW_PREFIX}/bin/compressdoc")
+  return if CREW_NOT_COMPRESS
+
+  # check for an lzip implementation, if none, use xz instead
+  if File.exist? "#{CREW_PREFIX}/bin/lzip"
+    compress = 'lzip'
+  else
+    compress = 'xz'
+  end
 
   if Dir.exist? dir
     system "find #{dir} -type f ! -perm -200 | xargs -r chmod u+w"
-    system "compressdoc --gzip -9 #{@short_verbose} #{dir}"
+    system "compressdoc --#{compress} -9 #{@short_verbose} #{@_short_verbose} #{dir}"
```

Comment:
Why do we need 2 short verboses?

Reply:
Because the program accepts verbose up to two times, to make it even more verbose.

Reply:
I'm confused, the `#{@_short_verbose}` variable doesn't even exist.

Reply:
Thanks for catching that, see 77624f5